deploy/csi-daemonset.yaml

```diff
@@ -30,8 +30,8 @@ spec:
       # the same.
       containers:
       - name: mayastor-csi
-        image: mayadata/mayastor:develop
-        imagePullPolicy: Always
+        image: mayadata/mayastor-csi:v0.8.1
```

Comment:
Rather than hardcoding a specific version here (and making merge conflicts more likely), consider using `sed` to replace the `:latest` tag with a specific version in the installation script, e.g.

```shell
(sed s/:develop/:v0.8.1/g < ./deploy/csi-daemonset.yaml) | kubectl apply -f -
```

Reply:
I see we're pulling the configuration files from the upstream reop in the installation script, so why this change?

Anyway, we can still use `sed` in the script and `curl` to get the configurations:

```shel
curl -sSfL https://raw.githubusercontent.com/openebs/mayastor/master/deploy/csi-daemonset.yaml | sed s/:develop/:v0.8.1/g | kubectl apply -f -
```

Reply:
the changes to deploy/csi-daemonset.yaml and deploy/mayastor-daemonset.yaml got included as a part of a merge. I'll revert these are not part of my changes.

Reply:
I merged with master instead of develop branch by habit.